PA_Feature_Detecting.md

```diff
@@ -68,3 +68,18 @@ of the web platform.
 This cannot be detected from the web platform. Instead, an ad auction can register both a `reserved.top_navigation_commit` and a `reserved.top_navigation`
 beacon, and then the ad frame can set the same automatic beacon data for both event types. The beacon destination server can then compare
 `top_navigation_commit` and `top_navigation` beacons, and filter out duplicate beacons that have the same exact data.
+
+## Increase in limit to number of component ads
```

Comment:
So given this doc apparently exists, I am not sure how much the main explainer should talk about it? Maybe I should just drop the code there?

Reply:
Probably fine to have it in both places for now, as this is the first time we're documenting queryFeatureSupport